server/src/main/java/org/cloudfoundry/identity/uaa/provider/IdentityProviderEndpoints.java

```diff
@@ -119,34 +135,56 @@ public ResponseEntity<IdentityProvider> createIdentityProvider(@RequestBody Iden
             samlConfigurator.validateSamlIdentityProviderDefinition(definition);
             body.setConfig(definition);
         }
+
```

Comment:
Are there any restrictions on the IdPs we want to allow mirroring for? For example, should we allow mirroring of IdPs of a certain type?

Reply:
In particular, should it be possible to mirror IdPs of type `UAA`?

Reply:
maybe ignore type UAA (or do the sync only for SAML and OAutth2/OIDC) because internal origins like UAA and LDAP are not really zone aware. They need extra config. Origin UAA is special in zone UAA and maybe not a good idea to sync with other zones

Reply:
In commit 862405896a1bcb8da76042569c71c72705c1ea35, I added an allowlist that contains SAML, OIDC and OAuth2.0. Are there any values I should add to/remove from there?

Reply:
I believe the 3 listed are all that is needed - SAML, OIDC and OAuth2.0